test/enforcer.test.ts

```diff
@@ -666,3 +666,11 @@ test('TestKeyGet2', async () => {
   expect(await e.enforce('alice', '/data')).toBe(false);
   expect(await e.enforce('alice', '/data/1')).toBe(true);
 });
+
+test('TestEnforceExWithDenyModel', async () => {
```

Comment:
Could you add more tests to cover EnforceEx with the supported policy effect?

Reply:
Add some more tests